### DISCUSSION TITLE
Welcome to youtube2Anki Discussions!

### DISCUSSION THREAD

**Discussion by dobladov** [dobladov/youtube2Anki#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NguyenNgoNghech** [comment]
Hi, i very like this extension, it help me a lot. But i see youtube just update new version about "open transcript" and now extension dont work with it. I see youtube update new IU for transcrift and may be extension can't read this. Can you just see a bit?. Thanks you so much
[](url
![Screenshot 2022-03-26 082127](https://user-images.githubusercontent.com/102395991/160219107-4a8402b3-036b-4cb2-b167-8951be142d7b.png)
)

**Reply by dobladov** [reply]
Hi @NguyenNgoNghech I will try to take a look at it over the weekend, I can see that it's broken, thanks for reporting it.

**Reply by dobladov** [reply]
@NguyenNgoNghech I summited a new version `1.0.2`, when Google finishes reviewing it should be published and working again.

**Reply by NguyenNgoNghech** [reply]
oh my god, you is best dev i know, thanks you soooo much. It help me a lot for learning.

**Reply by dobladov** [reply]
The new version should be available now (you can try updating the extension), please take a look and if something it's still not working, let me know.

Thanks for your feedback.
